pkg/analytics/track.go

```diff
@@ -49,7 +49,7 @@ const (
 	doctorEvent              = "Doctor"
 	buildEvent               = "Build"
 	buildTransientErrorEvent = "BuildTransientError"
-	deployEvent              = "Deploy"
+	deployEvent              = "OktetoDeploy"
```

Comment:
@rlamana this is the new event name. Between this change and the recent backend change we'll no longer get the `Deploy` event. Just confirming we are ok with this

Reply:
So, is this the event for the nested ones? Or in general? If we end up starting a new mixpanel project like we've been discussing, I want this event to still be `Deploy`. And if we end up using a prefix, I think all these events (UP, CONTEXT, etc..) should have the same prefix.